src/Tests/Microsoft.Diagnostics.Monitoring.Tool.FunctionalTests/Options/OptionsExtensions.cs

```diff
@@ -89,6 +89,19 @@ public static RootOptions EnableInProcessFeatures(this RootOptions options)
             return options;
         }
 
+        public static RootOptions SetExceptionFiltering(this RootOptions options, ExceptionsConfiguration configuration)
```

Comment:
nit: This method should only do enough to set the configuration elements to provide the filter configuration; it shouldn't enable in-process features and it shouldn't enable exceptions itself. Let the caller do those (and technically, only exceptions needs to be enabled, not all of in-process features).